flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/reconciler/diff/DiffResult.java

```diff
@@ -59,18 +63,48 @@ public String toString() {
             return "";
         }
 
-        final ToStringBuilder lhsBuilder =
-                new ToStringBuilder(left, ToStringStyle.SHORT_PREFIX_STYLE);
-        final ToStringBuilder rhsBuilder =
-                new ToStringBuilder(right, ToStringStyle.SHORT_PREFIX_STYLE);
+        final StringBuilder builder = new StringBuilder();
+        builder.append(left.getClass().getSimpleName()).append("[");
 
         diffList.forEach(
                 diff -> {
-                    lhsBuilder.append(diff.getFieldName(), diff.getLeft());
-                    rhsBuilder.append(diff.getFieldName(), diff.getRight());
+                    try {
```

Comment:
This part should be extracted to a method

Reply:
Which part do you think should be extracted to a method, and why?